themes/bootstrap3/templates/oai/home.phtml

```diff
@@ -3,198 +3,196 @@
-        <div class="col-sm-8">
-            <input type="hidden" name="verb" value="GetRecord"/>
-            <p class="help-block">Returns a single record.</p>
-            <div class="form-group">
-              <label for="GetRecord_identifier" class="col-sm-3 control-label"><?=$this->transEsc('Identifier')?>:</label>
-              <div class="col-sm-9">
-                <input id="GetRecord_identifier" type="text" name="identifier" class="form-control"/>
-              </div>
+    </div>
+  </form>
+  <hr/>
+  <a name="GetRecord"></a>
+  <h2>GetRecord</h2>
+  <form class="form-horizontal" method="get" action="<?=$baseUrl?>">
+    <div class="row">
+      <div class="col-sm-8">
+          <input type="hidden" name="verb" value="GetRecord"/>
+          <p class="help-block">Returns a single record.</p>
+          <div class="form-group">
+            <label for="GetRecord_identifier" class="col-sm-3 control-label"><?=$this->transEsc('Identifier')?>:</label>
+            <div class="col-sm-9">
+              <input id="GetRecord_identifier" type="text" name="identifier" class="form-control"/>
             </div>
-            <div class="form-group">
-              <label for="GetRecord_metadataPrefix" class="col-sm-3 control-label"><?=$this->transEsc('Metadata Token')?>:</label>
-              <div class="col-sm-9">
-                <input id="GetRecord_metadataPrefix" type="text" name="metadataPrefix" class="form-control"/>
-              </div>
+          </div>
+          <div class="form-group">
+            <label for="GetRecord_metadataPrefix" class="col-sm-3 control-label"><?=$this->transEsc('Metadata Token')?>:</label>
+            <div class="col-sm-9">
+              <input id="GetRecord_metadataPrefix" type="text" name="metadataPrefix" class="form-control"/>
             </div>
-        </div>
-        <div class="col-sm-4">
-          <input class="btn btn-default" type="submit" name="submit" value="<?=$this->transEsc('Go')?>"/>
-        </div>
+          </div>
+      </div>
+      <div class="col-sm-4">
+        <input class="btn btn-default" type="submit" name="submit" value="<?=$this->transEsc('Go')?>"/>
       </div>
-    </form>
-  </div>
-  <div class="col-sm-3">
-    <h2><?=$this->transEsc('Available Functionality') ?>:</h2>
-    <div class="list-group">
-      <a href="#Identify" class="list-group-item active">Identify</a>
-      <a href="#ListIdentifiers" class="list-group-item">ListIdentifiers</a>
-      <a href="#ListMetadataFormats" class="list-group-item">ListMetadataFormats</a>
-      <a href="#ListSets" class="list-group-item">ListSets</a>
-      <a href="#ListRecords" class="list-group-item">ListRecords</a>
-      <a href="#GetRecord" class="list-group-item">GetRecord</a>
     </div>
+  </form>
+</div>
+<div class="<?=$this->layoutClass('sidebar')?>">
+  <h2><?=$this->transEsc('Available Functionality') ?>:</h2>
+  <div class="list-group">
+    <a href="#Identify" class="list-group-item active">Identify</a>
```

Comment:
Not a problem specific to this PR (same behavior in both places), but why is this item flagged as active? There's no way to manipulate the menu, so it seems weird to highlight one item just because it happens to be at the top.